### ISSUE TITLE
Add indicator to episodes that the user has already listened to until the end

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#446]
Currently when a user listens all the way until the end of an episode, the episode cell does not indicate that it was listened to until the end. Some kind of indicator needs to be added to the TimeRemainingWidget to show "already listened" status.

**Comment by mitchdowney** [comment]
This issue is blocked by https://github.com/react-native-kit/react-native-track-player/issues/472

**Comment by mitchdowney** [comment]
This is basically a duplicate of https://github.com/podverse/podverse-rn/issues/461

Closing this issue.
